dbaas-cli/cmd/mongo/create.go

```diff
@@ -84,24 +86,28 @@ var createCmd = &cobra.Command{
 		defer tckr.Stop()
 		for range tckr.C {
 			cluster, err := dbaas.DescribeDB(instance)
-			if err != nil {
+			if err != nil && err != k8s.ErrOutOfMemory {
 				//dotPrinter.StopPrintDot("error")
 				//log.Error("check db: ", err)
 				continue
 			}
 			switch cluster.Status {
-			case "ready":
+			case structs.StateReady:
```

Comment:
"structs" is not the best package name. e.g. look at https://dave.cheney.net/practical-go/presentations/gophercon-singapore-2019.html#_avoid_package_names_like_base_common_or_util

From what I see the content of this package belongs to "github.com/Percona-Lab/percona-dbaas-cli/dbaas-lib"

Reply:
https://github.com/Percona-Lab/percona-dbaas-cli/pull/101/commits/0dccdc08eeca4fe39a3cbf9e7e558227fbe775c5